### ISSUE TITLE
Opengraph Output Too Large

### ISSUE THREAD

**Issue by DestroyerXyz** [vercel/next.js#59975]
### Link to the code that reproduces this issue

https://codesandbox.io/p/devbox/vercel-next-js-reproduction-template-go8s7s

### To Reproduce

1. Create an opengraph-image.tsx file and proceed with instructions in next docs.
2. Deploy to Vercel.

### Current vs. Expected behavior

Before next@14.0.5-canary.23, when @vercel/og was upgraded to 0.6.1, the same code produced an output within Vercel's limit for an edge function.
Now it exceeds it, the output seems excessively large, the only lib imported is next/og and the file consists of a minimal amount of code.

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.3.0: Fri Dec  1 03:16:57 PST 2023; root:xnu-10002.80.11~58/RELEASE_ARM64_T8103
Binaries:
  Node: 21.5.0
  npm: 10.2.4
  Yarn: 1.22.21
  pnpm: 8.12.1
Relevant Packages:
  next: 14.0.5-canary.28
  eslint-config-next: 14.0.4
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Not sure

<sub>[NEXT-1973](https://linear.app/vercel/issue/NEXT-1973/opengraph-output-too-large)</sub>

**Comment by JesseKoldewijn** [comment]
I noticed this as well after moving to the (currently) latest canary release. It seems like from one version to another the output of the OpenGraph functions I got in my project got drastically bigger.

I moved from `14.0.5-canary.22` to `14.0.5-canary.35` and ended up with the result of being unable to deploy as of now (on the latest canary release, mind you)

**Comment by huozhi** [comment]
Fixed by #60205, please upgrade to new canary release `14.0.5-canary.38`, thanks
